src/pages/Expenses.jsx

```diff
@@ -10,6 +18,22 @@ export const Expenses = () => {
     <>
       <Box bg={bgMain} w="100%" p={4}>
         <Flex bg={cardBg} direction="column" justify="center" align="center">
+          <Box bg={bgMain} w="100%" px={24} py={6}>
+            <HStack spacing={4}>
+              <Tag size="lg" variant="solid" colorScheme="teal">
+                <TagLabel>Food</TagLabel>
+                <TagCloseButton />
+              </Tag>
```

Comment:
create a separate component for the tag so that it can be reused and mapped

Reply:
done :)